snakemake/logging.py

```diff
@@ -393,7 +393,7 @@ def timestamp():
                     self.logger.warning(indent(msg["msg"]))
             elif level == "job_finished" and not self.quiet:
                 timestamp()
-                self.logger.info("Finished job {}.".format(msg["jobid"]))
+                self.logger.info("Finished job {}: {}.".format(msg["jobid"], msg["name"]))
```

Comment:
any reason not to use f-strings?

Reply:
Yes, Snakemake still support python 3.5. Once that is gone, I'd be happy to change all that. But I am still hesitating.